src/contracts/interfaces/IStakeRegistry.sol

```diff
@@ -126,26 +137,20 @@ interface IStakeRegistry is IRegistry {
      * bad `stakeHistoryIndex` can be supplied in order to obtain a response of 'false'.
      */
     function checkOperatorInactiveAtBlockNumber(
-        address operator,
+        bytes32 operatorId,
         uint256 blockNumber,
         uint8 quorumNumber,
         uint256 stakeHistoryIndex
         ) external view returns (bool);
 
-    /**
-     * @notice Returns the most recent stake weight for the `operator` for quorum `quorumNumber`
-     * @dev Function returns weight of **0** in the event that the operator has no stake history
-     */
-    function getCurrentOperatorStakeForQuorum(address operator, uint8 quorumNumber) external view returns (uint96);
-
-    /// @notice Returns the stake weight from the latest entry in `totalStakeHistory` for quorum `quorumNumber`.
-    function getCurrentTotalStakeForQuorum(uint8 quorumNumber) external view returns (uint96);
-
     /**
      * @notice Used for updating information on deposits of nodes.
-     * @param operatorIds are the ids of the nodes whose deposit information is getting updated
+     * @param operators are the addresses of the operators whose stake information is getting updated
+     * @param operatorIds are the ids of the operators whose stake information is getting updated
```

Comment:
is there a reason we need _both_ of these inputs here, but only `operatorId` elsewhere?

Reply:
because the stake is tied to the operator in eigenlayer, the storage of the AVS is tied to the operatorId

Reply:
OK, makes sense.